packages/applications/ssr/src/app/candidatures/corriger-par-lot/CorrigerCandidaturesParLot.action.ts

```diff
@@ -82,7 +82,12 @@ const action: FormAction<FormState, typeof schema> = async (_, { fichierCorrecti
     return {
       status: 'success',
       result: {
-        successCount: success,
+        message:
+          success === 0
+            ? ''
+            : success === 1
+              ? `${success} candidat corrigé`
+              : `${success} candidat(s) corrigé(s)`,
```

Comment:
Tu peux carrément mettre
```suggestion
              : `${success} candidats corrigés`,
```